README.md

```diff
@@ -62,31 +62,33 @@ $ git config --local gitsign.fulcio https://fulcio.example.com
+| GITSIGN_CREDENTIAL_CACHE  | ❌                 |                                  | Optional path to [gitsign-credential-cache](cmd/gitsign-credential-cache/README.md) socket.                                                                                                                                                |
+| GITSIGN_CONNECTOR_ID      | ✅                 |                                  | Optional Connector ID to auto-select to pre-select auth flow to use. For the public sigstore instance, valid values are:<br>- `https://github.com/login/oauth`<br>- `https://accounts.google.com`<br>- `https://login.microsoftonline.com` |
+| GITSIGN_FULCIO_URL        | ✅                 | https://fulcio.sigstore.dev      | Address of Fulcio server                                                                                                                                                                                                                   |
+| GITSIGN_LOG               | ❌                 |                                  | Path to log status output. Helpful for debugging when no TTY is available in the environment.                                                                                                                                              |
+| GITSIGN_OIDC_CLIENT_ID    | ✅                 | sigstore                         | OIDC client ID for application                                                                                                                                                                                                             |
+| GITSIGN_OIDC_ISSUER       | ✅                 | https://oauth2.sigstore.dev/auth | OIDC provider to be used to issue ID token                                                                                                                                                                                                 |
+| GITSIGN_OIDC_REDIRECT_URL | ✅                 |                                  | OIDC Redirect URL                                                                                                                                                                                                                          |
+| GITSIGN_REKOR_URL         | ✅                 | https://rekor.sigstore.dev       | Address of Rekor server                                                                                                                                                                                                                    |
+| GITSIGN_TIMESTAMP_URL     | ✅                 |                                  | Address of timestamping authority. If set, a trusted timestamp will be included in the signature.                                                                                                                                          |
```

Comment:
What about GITSIGN_TSA_URL ? I know naming is hard.

Reply:
Is this what cosign is using? 👀 I'd be happy to switch this for consistency (especially if both tools can use the `SIGSTORE_` variant to configure them simultaneously)

Reply:
If you look for consistency, I'll reuse  `--timestamp-cert-chain` `--timestamp-server-url`